gradle.properties

```
@@ -1,2 +1,2 @@
-version=7.0.0
-iexecCommonVersion=5.8.0
\ No newline at end of file
+version=7.1.0-rc1
```

```suggestion
version=7.1.0
```
Is `rc1` required here for the lib?